assignment_3.py

```diff
@@ -0,0 +1,120 @@
+def Psyco(input_student):
+    return input_student
+print(Psyco("pavi")) #pavi
+def ds(input_students):
+    return input_students + 15
+print(ds(15))  #25
+def ds(input_students):
+    return input_students + 'string'
+print(ds("pavi")) #pavistring
+def ds(input_students):
+    return input_students + 10
+try:

```

Comment:
You need to put try exception or any other business logics inside a function. you should not put it outside.